src/app/assets/languages/en.json

```diff
@@ -118,6 +104,24 @@
       "modal-enable-button": "Turn on rebalancing",
       "modal-cancel-button": "Cancel",
       "modal-description": "Temporarily disable only to avoid repeated cluster rebalancing operations when restarting a node or adding multiple nodes. Always enable cluster rebalancing as soon as possible to prevent data loss."
+    },
+    "counters": {
+      "strong-counters": "Strong counters",
+      "weak-counters": "Weak counters",
+      "strong": "Strong",
+      "weak": "Weak",
+      "counter-name": "Name",
+      "current-value": "Current value",
+      "initial-value": "Initial value",
+      "lower-bound": "Lower bound:",
+      "upper-bound": "Upper bound:",
+      "concurrency-level": "Concurrency level:",
+      "storage": "Storage",
+      "counter-configuration": "Configuration",
+      "counter-type": "Counter type",
+      "no-counters-status": "No counters yet",
+      "no-counters-body": "Select \"Create counter\" or you can also create counters from the CLI and remote client.",
+      "no-filtered-counter-body": "No counters match the filter."
```

Comment:
@dvagnero we need a body saying "The filters you have chosen doesn't have any counters" or like that. Please check if this work or we need to change.

Reply:
@dpanshug I'd say "No counters match your filter criteria. Click \"Create counter\" and configure a counter."
Do you plan to add this message to filters for Caches, Tasks etc?

Reply:
Yes we may add this similar for other things also